vmcnet/mcmc/position_amplitude_core.py

```diff
@@ -122,6 +122,8 @@ def distribute_position_amplitude_data(
 def make_position_amplitude_gaussian_proposal(
     model_apply: ModelApply[P],
     get_std_move: Callable[[PositionAmplitudeData], jnp.float32],
+    discrete: bool=False,
+    cyclic: int=0,
```

Comment:
Typing-wise, this should probably be `Array`, since `x_1 % x_2` is defined for arrays `x_1` and `x_2` (https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.mod.html). This will also give us the flexibility to explore rectangular Hubbard models, etc.